packages/transformer/src/IModelTransformer.ts

```diff
@@ -1464,12 +1469,21 @@ export class IModelTransformer extends IModelExportHandler {
     }
   }
 
+  /**
+   * Remember all target element ids that were newly inserted during current change processing.
+   */
+  public override async preDeleteElements(): Promise<void> {
+    this.exporter.sourceDbChanges?.element.insertIds.forEach((insertedSourceElementId) => this._insertedTargetElementIds.add(this.context.findTargetElementId(insertedSourceElementId)));
```

Comment:
why introduce a new handler for this? Why not just a new private method in the `IModelTransformer` itself? I am hesitant to add new (even internal) handler methods, and I of the reasons I can think of that you have, I think there are better alternatives

Reply:
needed that in order to access element remap table from exporter. Deleted this one and exposed `findTargetElementId` instead.

Reply:
resolving this as outdated, my new review addresses any concerns I have